### ISSUE TITLE
Add documentation for `run_northstar.txt`

### ISSUE THREAD

**Issue by GeckoEidechse** [R2Northstar/NorthstarWiki#87]
When doing some work on Launcher I came across this commit:
https://github.com/R2Northstar/NorthstarLauncher/commit/d2ee389192aa425ef9c81b2c3367ffb0de6976d0

which comes from this PR
https://github.com/R2Northstar/NorthstarLauncher/pull/19

but was never properly documented.

I'm not sure if it's even still in use.

**Comment by GeckoEidechse** [comment]
@p0358 the mentioned PR is yours, maybe you could give some more info <3

**Comment by p0358** [comment]
It was meant to be an alternative to `-northstar` launch option in case someone cannot set the launch options for whatever reason or doesn't want to. The existence of this file implies you intend to run Northstar, unless the contents of this file are (starting with) `0`. If the file has `0` in it that means you do **not** want to run Northstar on the other hand, which is equivalent to `-vanilla` option in NorthstarLauncher.exe, which is the opposite to `-northstar` in Titanfall2.exe, which means the Northstar will not be loaded even if you use NorthstarLauncher.exe.

In short:
1. `run_northstar.txt` exists, any content (for example `1`) = Northstar is always loaded 
2. `run_northstar.txt` exists with content `0` = Northstar is not loaded

With:
1. being mostly relevant for Titanfall2.exe (equivalent of `-northstar`)
2. being mostly relevant for NorthstarLauncher.exe (equivalent of `-vanilla`)

**Comment by p0358** [comment]
It wasn't in the end documented or widely recommended as people turned out to generally have enough brain cells to add the launch options
